### PR TITLE
Spawn enemies per waves

### PR DESCRIPTION
**Pull request by Anisetobegan**

### CODE REVIEW

File: Assets/Scripts/EnemySpawner.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        Enemies newEnemy = Instantiate(enemiesToSpawn[i], new Vector3(UnityEngine.Random.Range(minOffset, maxOffset),
+            transform.position.y, UnityEngine.Random.Range(minOffset, maxOffset)), transform.rotation);
+
+        enemiesAlive.Add(newEnemy);
+    }
+
+    public int GetEnemiesKilled()
+    {
+        return enemiesKilled;
+    }
```
Comment (morathias) [inline Assets/Scripts/EnemySpawner.cs]: Make this a Getter

File: Assets/Scripts/EnemySpawner.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+    public int GetMaxEnemies()
+    { 
+        return maxEnemies;
+    }
+
+    public void SetMaxEnemies(int enemiesToAdd)
+    {
+        maxEnemies += enemiesToAdd;
+    }
```
Comment (morathias) [inline Assets/Scripts/EnemySpawner.cs:76]: Make this a getter and setter
public int MaxEnemies {get => maxEnemies; set => maxEnemies = value;}

File: Assets/Scripts/GameManager.cs [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    public void Lose()
+    {
+
+    }
+
+    public IEnumerator StartNewWave()
+    {
+        Debug.Log("Starting wave " + wave.ToString());
```
Comment (morathias) [inline Assets/Scripts/GameManager.cs]: Remove Debug log

**Review by morathias — CHANGES_REQUESTED** [review]

File: Assets/Scripts/EnemySpawner.cs (hunk 1 above)
Comment (Anisetobegan) [reply Assets/Scripts/EnemySpawner.cs]: Done

File: Assets/Scripts/EnemySpawner.cs (hunk 2 above)
Comment (Anisetobegan) [reply Assets/Scripts/EnemySpawner.cs:76]: Done

File: Assets/Scripts/GameManager.cs (hunk 3 above)
Comment (Anisetobegan) [reply Assets/Scripts/GameManager.cs]: Done

**Review by morathias — APPROVED** [review]

**Pull request merged**